performance/run.js

```diff
@@ -0,0 +1,12 @@
+const { execSync } = require('child_process');
+
+if(process.platform === "darwin") {
+    execSync('docker run --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io:12.2.0 --config=config/config-chrome.json http://host.docker.internal:8080/performance/performance.html', {stdio: 'inherit', cwd: __dirname});
```

Comment:
en dit faalt nu als ge ergens onder bepaalde thresholds zit? (zie vraag naar documentatie hierboven)

Reply:
Yes, tresholds zijn bepaalde in de budget.json.